packages/app/navigation/linking.ts

```diff
@@ -0,0 +1,141 @@
+                parse: parsePathParams('groupId', 'channelId'),
+              },
+              GroupMeta: {
+                path: 'group/:groupId/meta',
+                parse: parsePathParams('groupId'),
+              },
+              GroupMembers: {
+                path: 'group/:groupId/members',
+                parse: parsePathParams('groupId'),
+              },
+              ManageChannels: {
+                path: 'group/:groupId/manage-channels',
+                parse: parsePathParams('groupId'),
+              },
+              Privacy: {
+                path: 'group/:groupId/privacy',
+                parse: parsePathParams('groupId'),
+              },
+              GroupRoles: {
+                path: 'group/:groupId/roles',
+                parse: parsePathParams('groupId'),
+              },
+            },
+          },
+          AppSettings: 'app-settings',
+          FeatureFlags: 'feature-flags',
+          ManageAccount: 'manage-account',
+          BlockedUsers: 'blocked-users',
+          WompWomp: 'report-bug',
+          AppInfo: 'app-info',
+          PushNotificationSettings: 'push-notification-settings',
+        },
+      },
+    },
+  },
+});
+
+export const getDesktopLinkingConfig = (
+  mode: string
+): LinkingOptions<DesktopBasePathStackParamList> => ({
+  prefixes: [],
+  config: {
+    screens: {
+      Root: {
+        path: basePathForMode(mode),
+        initialRouteName: 'Home',
+        screens: {
+          Activity: 'activity',
+          Profile: 'profile',
+          Home: {
+            screens: {
+              ChatList: '',
+              GroupChannels: 'group/:groupId',
+              Channel: {
+                initialRouteName: 'ChannelRoot',
+                path: 'group/:groupId/channel/:channelId',
+                parse: parsePathParams('channelId', 'groupId'),
+                screens: {
+                  ChannelRoot: {
+                    path: '',
+                  },
+                  UserProfile: {
+                    path: 'profile/:userId',
+                    parse: parsePathParams('userId'),
+                    exact: true,
+                  },
+                  EditProfile: {
+                    path: 'profile/:userId/edit',
+                    parse: parsePathParams('userId'),
+                    exact: true,
+                  },
+                  ChannelMembers: {
+                    path: 'group/:groupId/channel/:channelId/members',
+                    parse: parsePathParams('channelId'),
+                    exact: true,
+                  },
+                  ChannelMeta: {
+                    path: 'group/:groupId/channel/:channelId/meta',
+                    parse: parsePathParams('channelId'),
+                    exact: true,
+                  },
+                  GroupSettings: {},
+                  ChannelSearch: {},
+                  Post: {
+                    path: 'post/:postId',
```

Comment:
Still trying to figure out what's happening here, but I seem to occasionally get a `:postId` of`undefined` and then that path component gets stuck in my URL bricking any other channels I try to navigate to

Reply:
yep, I see this when starting/opening a thread, URL path looks like this:
`https://localhost:3000/apps/tm-alpha/group/~rabpur-datpel-finned-palmer%2Ftesting-group-admin/channel/chat%2F~rabpur-datpel-finned-palmer%2Fwelcome-7767/post/undefined?post=%5Bobject%20Object%5D`

Reply:
Yeah, I see this too, fix incoming.

Reply:
looks like navigate to post still passes the whole post as a param:
```
  const navigateToPost = useCallback(
    (post: db.Post) => {
      navigation.push('Post', { post });
    },
    [navigation]
  );
```

Reply:
Should be fixed